components/simple_tooltip.tsx

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
```

Comment:
Should it be in the widgets?

Reply:
I guess it could be, but I thought it was a bit more generic that what most of the widgets+stories are.

Reply:
I think, for the widgets, the simpler the better, widgets shouldn't have anything specially complex.

Reply:
moved to widgets ✓